packages/discord.js/src/structures/PartialGroupDMChannel.js

```diff
@@ -11,6 +11,9 @@ class PartialGroupDMChannel extends BaseChannel {
   constructor(client, data) {
     super(client, data);
 
+    // No flags are present when fetching partial group DM channels.
+    delete this.flags;
```

Comment:
How does this show up in the docs? Perhaps it should just be set to a predefined value with a note that it doesn't exist?